src/main/java/eisenwave/elytra/command/ElytraToggleCommand.java

```diff
@@ -127,31 +133,54 @@ public boolean onCommand(final CommandSender commandSender, final Command comman
                 this.plugin.getMessenger().sendMessage(commandSender, "firework-disabled", true);
             }
         }
+        if (args[0].equalsIgnoreCase("personalchargeup")) {
```

Comment:
This probably should have a permission check as well, and should be added to the `plugin.yml` as a false-by-default permission. In order for people to be able to update to this, they should be able to decide if their players should have access to this command.